Index: gpac-0.5.0.svn5178/src/scenegraph/html5_mse_smjs.c
===================================================================
--- gpac-0.5.0.svn5178.orig/src/scenegraph/html5_mse_smjs.c
+++ gpac-0.5.0.svn5178/src/scenegraph/html5_mse_smjs.c
@@ -761,7 +761,8 @@ void gf_arraybuffer_del(GF_HTML_ArrayBuf
     if (buffer) {
         if (del_js && buffer->_this) {
 			/*if we have a JS object, make sure the parent XHR doesn't point to us ...*/
-			JSObject *xhro = JS_GetParent(buffer->c, buffer->_this); 
+			/*JSObject *xhro = JS_GetParent(buffer->c, buffer->_this); */
+			JSObject *xhro = JS_GetParent(buffer->_this);
 			if (xhro) {
 				void *xhr_ctx = SMJS_GET_PRIVATE(buffer->c, xhro);
 				xhr_del_array_buffer(xhr_ctx);
