diff -ur gpac-0.5.0.svn5456.orig/Makefile gpac-0.5.0.svn5456/Makefile
--- gpac-0.5.0.svn5456.orig/Makefile	2014-09-24 15:07:49.000000000 -0400
+++ gpac-0.5.0.svn5456/Makefile	2014-10-04 20:55:50.697803206 -0400
@@ -67,6 +67,7 @@
 install:
 	$(INSTALL) -d "$(DESTDIR)$(prefix)"
 	$(INSTALL) -d "$(DESTDIR)$(prefix)/$(libdir)"
+	$(INSTALL) -d "$(DESTDIR)$(prefix)/$(libdir)/pkgconfig"
 	$(INSTALL) -d "$(DESTDIR)$(prefix)/bin"
 ifeq ($(DISABLE_ISOFF), no) 
 ifeq ($(CONFIG_LINUX), yes)
