diff -ur gpac-0.5.0.svn5456.orig/modules/amr_float_dec/amr_float_dec.c gpac-0.5.0.svn5456/modules/amr_float_dec/amr_float_dec.c
--- gpac-0.5.0.svn5456.orig/modules/amr_float_dec/amr_float_dec.c	2014-04-26 16:08:43.000000000 -0400
+++ gpac-0.5.0.svn5456/modules/amr_float_dec/amr_float_dec.c	2014-10-04 20:54:55.285877782 -0400
@@ -46,12 +46,12 @@
 
 #ifdef GPAC_HAS_AMR_FT
 /*AMR WB*/
-#include "amr_nb_ft/interf_dec.h"
+#include <amrnb/interf_dec.h>
 #endif
 
 #ifdef GPAC_HAS_AMR_FT_WB
 /*AMR WB*/
-#include "amr_wb_ft/dec_if.h"
+#include <amrwb/dec_if.h>
 #endif
 
 #endif
diff -ur gpac-0.5.0.svn5456.orig/modules/amr_float_dec/Makefile gpac-0.5.0.svn5456/modules/amr_float_dec/Makefile
--- gpac-0.5.0.svn5456.orig/modules/amr_float_dec/Makefile	2014-01-24 14:44:54.000000000 -0500
+++ gpac-0.5.0.svn5456/modules/amr_float_dec/Makefile	2014-10-04 20:54:55.285877782 -0400
@@ -18,19 +18,19 @@
 #common obj
 OBJS=../amr_dec/amr_in.o amr_float_dec.o
 
+AMR_LIBS=
+
 #AMR NB obj
 CFLAGS+=-DGPAC_HAS_AMR_FT
 ifeq ($(CONFIG_AMR_NB_FT), yes)
-OBJS+=./amr_nb_ft/interf_dec.o ./amr_nb_ft/interf_enc.o ./amr_nb_ft/sp_dec.o ./amr_nb_ft/sp_enc.o
+AMR_LIBS+=-lamrnb
 endif
 
 
 #AMR WB obj
 CFLAGS+=-DGPAC_HAS_AMR_FT_WB
 ifeq ($(CONFIG_AMR_WB_FT), yes)
-OBJS+=./amr_wb_ft/dec_acelp.o ./amr_wb_ft/dec_dtx.o ./amr_wb_ft/dec_gain.o ./amr_wb_ft/dec_if.o ./amr_wb_ft/dec_lpc.o ./amr_wb_ft/dec_main.o \
-	./amr_wb_ft/dec_rom.o ./amr_wb_ft/dec_util.o ./amr_wb_ft/enc_acelp.o ./amr_wb_ft/enc_dtx.o ./amr_wb_ft/enc_gain.o ./amr_wb_ft/enc_if.o \
-	./amr_wb_ft/enc_lpc.o ./amr_wb_ft/enc_main.o ./amr_wb_ft/enc_rom.o ./amr_wb_ft/enc_util.o ./amr_wb_ft/if_rom.o 
+AMR_LIBS+=-lamrwb
 endif
 
 
@@ -45,7 +45,7 @@
 
 
 $(LIB): $(OBJS)
-	$(CC) $(SHFLAGS) $(LDFLAGS) -o ../../bin/gcc/$@ $(OBJS) $(EXTRALIBS) -L../../bin/gcc -lgpac
+	$(CC) $(SHFLAGS) $(LDFLAGS) -o ../../bin/gcc/$@ $(OBJS) $(EXTRALIBS) -L../../bin/gcc -lgpac $(AMR_LIBS)
 
 clean: 
 	rm -f $(OBJS) ../../bin/gcc/$(LIB)
