Index: gpac-0.5.0.svn5178/applications/mp4box/main.c
===================================================================
--- gpac-0.5.0.svn5178.orig/applications/mp4box/main.c
+++ gpac-0.5.0.svn5178/applications/mp4box/main.c
@@ -1388,7 +1388,7 @@ GF_DashSegmenterInput *set_dash_input(GF
 				if (strlen(opts+6) > 199) {
 					GF_LOG(GF_LOG_ERROR, GF_LOG_DASH, ("[DASH] XLink cannot exceed 99 characters in MP4Box, truncating ...\n"));
 				}
-				strncpy(di->xlink, opts+6, 199);
+				strncpy(di->xlink, opts+6, 99);
 			}
 			
 			if (!sep) break;
