diff -ur gpac-0.5.0.svn5456.orig/applications/Makefile gpac-0.5.0.svn5456/applications/Makefile
--- gpac-0.5.0.svn5456.orig/applications/Makefile	2014-08-08 07:30:37.000000000 -0400
+++ gpac-0.5.0.svn5456/applications/Makefile	2014-10-04 20:38:57.296656928 -0400
@@ -27,15 +27,15 @@
 INSTDIRS=mp4client
 ifeq ($(CONFIG_XUL),no)
 else
-#INSTDIRS+=osmozilla
-#APPDIRS+=osmozilla
+INSTDIRS+=osmozilla
+APPDIRS+=osmozilla
 endif
 
 #disable due to version incompatibilities
 ifeq ($(USE_WXWIDGETS), yes)
-#APPDIRS+=osmo4_wx
-#V4STUDIODIR=V4Studio
-#INSTDIRS+=osmo4_wx
+APPDIRS+=osmo4_wx
+V4STUDIODIR=V4Studio
+INSTDIRS+=osmo4_wx
 endif
 
 ALLDIRS=$(APPDIRS)
diff -ur gpac-0.5.0.svn5456.orig/applications/osmo4_wx/Makefile gpac-0.5.0.svn5456/applications/osmo4_wx/Makefile
--- gpac-0.5.0.svn5456.orig/applications/osmo4_wx/Makefile	2014-01-24 14:44:54.000000000 -0500
+++ gpac-0.5.0.svn5456/applications/osmo4_wx/Makefile	2014-10-04 20:38:57.296656928 -0400
@@ -46,7 +46,7 @@
 all: $(PROG)
 
 Osmo4$(EXE): $(OBJS)
-	$(CC) $(LDFLAGS) -o ../../bin/gcc/$@ $(OBJS) -L../../bin/gcc -lgpac $(WX_LFLAGS) $(LINKFLAGS)
+	$(CC) $(LDFLAGS) -o ../../bin/gcc/$@ $(OBJS) -L../../bin/gcc -lgpac -lX11 -lgtk-x11-2.0 -lgdk-x11-2.0 -lstdc++ -lm $(WX_LFLAGS) $(LINKFLAGS)
 
 clean: 
 	rm -f $(OBJS) ../../bin/gcc/$(PROG)
diff -ur gpac-0.5.0.svn5456.orig/applications/osmozilla/osmo_npapi.cpp gpac-0.5.0.svn5456/applications/osmozilla/osmo_npapi.cpp
--- gpac-0.5.0.svn5456.orig/applications/osmozilla/osmo_npapi.cpp	2014-04-26 16:08:43.000000000 -0400
+++ gpac-0.5.0.svn5456/applications/osmozilla/osmo_npapi.cpp	2014-10-04 20:40:21.768585241 -0400
@@ -372,10 +372,11 @@
 	/* explicit plugin call */ \
 	"application/x-gpac::GPAC plugin;" \
  
-char * NP_GetMIMEDescription(void)
+/*char * NP_GetMIMEDescription(void)
 {
 	return (char *) GPAC_PLUGIN_MIMETYPES;
 }
+*/
 
 
 NPError NP_GetValue(void *future, NPPVariable aVariable, void *aValue)
