diff -urN gpac-0.5.0.svn4192.orig/applications/generators/SVG/Makefile gpac-0.5.0.svn4192/applications/generators/SVG/Makefile
--- gpac-0.5.0.svn4192.orig/applications/generators/SVG/Makefile	2012-10-31 04:14:54.011804771 +0800
+++ gpac-0.5.0.svn4192/applications/generators/SVG/Makefile	2012-10-31 04:21:04.182859948 +0800
@@ -35,7 +35,7 @@
 all: $(PROG)
 
 SVGGen$(EXE): $(OBJS) 
-	$(CC) $(LDFLAGS) -o $@ $(OBJS) $(XML2_LIBS) $(EXTRALIBS) -L../../../bin/gcc -L../../../extra_lib/lib/gcc -lgpac -lz
+	$(CC) $(LDFLAGS) -o $@ $(OBJS) $(XML2_LIBS) $(EXTRALIBS) -L../../../bin/gcc -L../../../extra_lib/lib/gcc -lgpac -lz -lxml2
 
 
 %.o: %.c
